troposphere/static/js/actions/instance/reboot.js

```diff
@@ -28,7 +28,7 @@ define(function (require) {
           attrs: {action: "reboot", reboot_type: params.reboot_type}
       }).done(function(){
         instance.set({
-          state: new InstanceState({status_raw: "active - rebooting"})
+          state: instanceState
```

Comment:
Wow! So this was creating `instanceState` on line 20 and then creating a new model with the same state and passing it on line 31. 

:recycle: thank you for cleaning up!